### PR TITLE
[14.0][IMP] sale_discount_display_amount: Adds hook for depends

### PR DESCRIPTION
**Pull request by rousseldenis**
Allows to override the different amount computations.

@chafique-delli @simahawk

### CODE REVIEW

File: sale_discount_display_amount/models/sale_order.py [hunk 1]
```diff
@@ -21,7 +21,11 @@ class SaleOrder(models.Model):
         store=True,
     )
 
-    @api.depends("order_line.discount_total", "order_line.price_total_no_discount")
+    @api.model
+    def _get_compute_discount_total_domain(self):
```
Comment (simahawk) [inline sale_discount_display_amount/models/sale_order.py]:
```suggestion
    def _compute_discount_total_depends(self):
```

File: sale_discount_display_amount/models/sale_order_line.py [hunk 2]
```diff
@@ -41,8 +41,12 @@ def _update_discount_display_fields(self):
                 }
             )
 
-    @api.depends("product_uom_qty", "discount", "price_unit", "tax_id")
+    @api.model
+    def _get_compute_amount_domain(self):
```
Comment (simahawk) [inline sale_discount_display_amount/models/sale_order_line.py]:
```suggestion
    def _compute_amount_depends(self):
```

**Review by simahawk — CHANGES_REQUESTED** [review]

**Review by simahawk — APPROVED** [review]
not blocking indeed. Pre-approving anyway ;)

**Comment by rousseldenis** [comment]
@simahawk Done

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**